pom.xml

```diff
@@ -271,6 +272,16 @@
                <enabled>false</enabled>
             </snapshots>
         </repository>
+
+        <!--
+        maven-dependency-plugin:3.1.2 seems to have updated HTTP repository access behavior.
```

Comment:
error was in `druid-thrift-extensions`:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.2.0:go-offline (default-cli) on project druid-thrift-extensions: org.eclipse.aether.resolution.DependencyResolutionException: Failed to collect dependencies at com.twitter.elephantbird:elephant-bird-core:jar:4.17 -> com.hadoop.gplcompression:hadoop-lzo:jar:0.4.19: Failed to read artifact descriptor for com.hadoop.gplcompression:hadoop-lzo:jar:0.4.19: Could not transfer artifact com.hadoop.gplcompression:hadoop-lzo:pom:0.4.19 from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [twitter (http://maven.twttr.com, default, releases+snapshots)] -> [Help 1]
```

* this seems to be deliberately excluded from the pom [here](https://github.com/apache/druid/blob/24a1fafaa745fb93bf1522490c095cd689fd9996/extensions-contrib/thrift-extensions/pom.xml#L83-L86) 
* and its documented to download it separately [here](https://github.com/apache/druid/blob/master/docs/development/extensions-contrib/thrift.md#lzo-support)
* at the same these artifacts are 10+ year old [ones](https://mvnrepository.com/artifact/com.hadoop.gplcompression)...I don't think we want them in the base build 

I'm not sure why `go-offline` still want to read that excluded artifact...and how it did that before...

but instead of adding this repo to the main pom; can we only add it for the `extensions-contrib/thrift-extensions` module?

Reply:
> but instead of adding this repo to the main pom; can we only add it for the extensions-contrib/thrift-extensions module?

I tried that originally. If you look at these 2 commits:

<img width="886" alt="image" src="https://github.com/user-attachments/assets/8976d3af-d6ff-42de-bcbb-02472bc6565d">

[First commit](https://github.com/apache/druid/pull/17466/commits/ea17aa1c463d4cc6e0b6a6ed4b2ba92726251ca7) added it to the `thrift-extensions` module, but that still caused the same error: [CI run for that commit](https://github.com/apache/druid/actions/runs/11782156018/job/32816565859)

[Second commit](https://github.com/apache/druid/pull/17466/commits/6d127c1e860c0f6751efb18621eff1d938dce087) moved it to the parent pom, which fixed the issue.

Thoughts?

Reply:
I feel like we could place `elephantbird` under a profile - and the [single usage](https://github.com/apache/druid/blob/24a1fafaa745fb93bf1522490c095cd689fd9996/extensions-contrib/thrift-extensions/src/main/java/org/apache/druid/data/input/thrift/ThriftInputRowParser.java#L135-L137) of `ThriftWritable` could be done via reflection and forget all that repo/etc

Reply:
I'm not sure if  this feature is even being used:
`elephant-bird` didn't had any releases for quite a few years now...
on the github page no real person opened a PR in the last 2 years or so ....last reported issues is also 2 years old..
I doubt its being used in production - what do you think about removing it?

Reply:
> I doubt its being used in production - what do you think about removing it?

Removing what exactly? "LZO support" from the thrift extension? 😅 

I don't have much context on this part, so can't comment. Regardless, this would require a discussion with the wider team in my opinion.

Reply:
lets leave it then for now...